### PR TITLE
Option information

### PR DESCRIPTION
**Pull request by vadigerman**
### Checklist
- [x] I've run `mvn test` from the root directory to see all new and existing tests pass
- [x] I've followed code style and run and ensured the code style is valid
- [ ] I've created new tests if necessary.

### Motivation and Context
https://github.com/ctco-dev/feedback-mood-rate/issues/24
https://github.com/ctco-dev/feedback-mood-rate/issues/21
https://github.com/ctco-dev/feedback-mood-rate/issues/15

### Description
Page "Statistics" opens only for admins, return eventDtoList.

### CODE REVIEW

File: src/main/java/lv/ctco/javaschool/vote/boundary/VoteApi.java [hunk 1]
```diff
@@ -3,10 +3,7 @@
 import lv.ctco.javaschool.auth.control.UserStore;
 import lv.ctco.javaschool.auth.entity.domain.User;
 import lv.ctco.javaschool.vote.control.VoteStore;
-import lv.ctco.javaschool.vote.entity.EventType;
-import lv.ctco.javaschool.vote.entity.Vote;
-import lv.ctco.javaschool.vote.entity.VoteDto;
-import lv.ctco.javaschool.vote.entity.VoteStatus;
+import lv.ctco.javaschool.vote.entity.*;
```
Comment (St-Filer) [inline src/main/java/lv/ctco/javaschool/vote/boundary/VoteApi.java]: Please avoid asterisk imports

File: src/main/java/lv/ctco/javaschool/vote/entity/Event.java [hunk 2]
```diff
@@ -0,0 +1,54 @@
+package lv.ctco.javaschool.vote.entity;
+
+import lv.ctco.javaschool.auth.entity.domain.User;
+import javax.persistence.*;
```
Comment (St-Filer) [inline src/main/java/lv/ctco/javaschool/vote/entity/Event.java]: Asterisk import

File: src/main/webapp/app/option.jsp [hunk 3]
```diff
@@ -3,13 +3,22 @@
...
+    <link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css">
 </head>
-<body>
-<button type="button" onclick="handlerEvent('DAY')">Day</button>
-<button type="button" onclick="handlerEvent('EVENT')">Event</button>
+<body onload="onloadHandler()">
+<h2>You mood in:</h2>
+<p>
+    <button type="button" onclick="handlerEvent('DAY')">Day</button>
+    <span id="date"></span></br></br>
```
Comment (St-Filer) [inline src/main/webapp/app/option.jsp]: Please replace closing `</br>` tag with correct one: `<br/>`

**Comment by vadigerman** [comment]
some fixes and merge conflict

**Pull request merged**